<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CafeStaffWindow</class>
 <widget class="QMainWindow" name="CafeStaffWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cafe Staff Portal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>My Workslots</string>
     </attribute>
     <widget class="QTableWidget" name="assignedTable">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>40</y>
        <width>391</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>86</width>
        <height>18</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Workslots:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="workslotText">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>1 / 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>185</width>
        <height>97</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Full Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="firstNameText"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Role: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="roleCombo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>My maximum work slots:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="maxSlotsBox">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="editInfoButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>120</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Changes</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Available Workslots</string>
     </attribute>
     <widget class="QPushButton" name="bidButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Bid</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Submitted Bids</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="submittedTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Available Workslots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="availableTable"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>My Bids</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Pending Bids</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="pendingTable"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Approved Bids</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="approvedTable">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Rejected Bids</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="rejectedTable">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="CafeResources.qrc">:/content/Copy of Untitled.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="CafeResources.qrc"/>
 </resources>
 <connections/>
</ui>
